doc/dev/cephfs-reclaim.rst

```diff
@@ -0,0 +1,71 @@
+CephFS Reclaim Interface
+========================
+
+Introduction
+------------
+NFS servers typically do not track ephemeral state on stable storage. If the
+NFS server is restarted, then it will be resurrected with no ephemeral state,
+and the NFS clients are expected to send requests to reclaim what state they
+held during a grace period.
+
+In order to support this use-case, libcephfs has grown several functions that
+allow a client that has been stopped and restarted to destroy or reclaim state
+held by a previous incarnation of itself.
+
+Setting the UUID
+----------------
+In order to properly reset or reclaim against the old session, we need a way
+to identify the old session. This done by setting a unique opaque value on
+the session using **ceph_set_uuid()**. The uuid value can be any string and
+is treated as opaque by the client. This should be done *after* any reclaim
+has been performed.
```

Comment:
uuid can only be set on new mount. reclaim inherit old session's uuid. Otherwise, looks good